### ISSUE TITLE
Can't craft chest and barrel variants

### ISSUE THREAD

**Issue by EastCoastCinnamonToast** [svenhjol/Charm#839]
Anytime I try to make the warped, azalea, acacia, and crimson variants of chests and barrels it gives me the vanilla version.

**Comment by verdantspell** [comment]
Do you have the chests enabled in the config?

**Comment by EastCoastCinnamonToast** [comment]
Yes. Although I just realized this was for 1.18 and not 1.19. That's my fault for not posting enough attention.

**Comment by adroidian** [comment]
I'm having this issue but only with barrels, including birch as well.  It makes a standard barrel. I'm playing 1.19
